subprojects/gradle/build-checks/src/main/kotlin/com/avito/android/build_checks/internal/CheckGradleDaemonTask.kt

```diff
@@ -3,15 +3,19 @@ package com.avito.android.build_checks.internal
 import com.avito.android.build_checks.RootProjectChecksExtension
 import org.gradle.api.DefaultTask
 import org.gradle.api.GradleException
-import org.gradle.api.Project
+import org.gradle.api.provider.Property
+import org.gradle.api.tasks.Input
 import org.gradle.api.tasks.TaskAction
 import java.io.File
 import java.util.Properties
 
 internal abstract class CheckGradleDaemonTask : DefaultTask() {
```

Comment:
We could remove this check. The implementation is useless with convention plugins.
Need to redesign it completely.

Reply:
will remove